### ISSUE TITLE
Need Camera Zoom feature.

### ISSUE THREAD

**Issue by hakuna0829** [zxing-js/library#328]
Hello.

Hope you guys are fine.
This lib is so great.
Btw, I have one trouble using this one.
Sometimes, I need to capture small size QRcode like 1.2cm x 1.2cm , and this lib doesn't work.
If it provides zoom in, then it would be great.
How can I add this zoom functionality using this lib?

If you have good idea to customize it, let me know soon.

Thanks.

**Comment by graemesmithmedkeeper** [comment]
I would like to see this too.

I wonder if there is any way around this to apply some cropping to the video feed with JS prior to ZXing processing it?

**Comment by odahcam** [comment]
Yes, there is, please search the issues and you will find a lot of info there. I sure need to document this when I got some time. 

Although there is a way to crop, I find it interesting to use native APIs to work with the native camera zoom, this is certainly something to look for as a feature.

**Comment by odahcam** [comment]
@graemesmithmedkeeper I remember to discuss cropping in #39 and #305 . Hope it helps.

**Comment by graemesmithmedkeeper** [comment]
Thanks, appreciate it! I sort of eventually figured it out myself and did I think something almost exactly like what this comment describes https://github.com/zxing-js/library/issues/305#issuecomment-653580507

I agree that some documentation improvements would be good. I really like this library (it's _really_ good as a totally free and extensible alternative to commercial libraries like Scandit and Dynamsoft). Documentation is a bit of a weak point but all it really needs is some attention from maintainers interested in writing about how it works.

**Comment by odahcam** [comment]
Yeah, documentation is really hard to do when your team is the size of 2 developers and both very busy. So  when we have time we choose to implement functionality and bugfixes prioritizing the existing users. That's a shame, unfortunately, but will get to the documentation point.
